src/css/map-view.css

```diff
@@ -1410,3 +1410,53 @@ other class: .ui-slider-range */
     }
   }
 }
+
+/** START - Expansion panel View. */
+.expansion-panel {
+	border: solid var(--portal-col-bkg-active);
```

Comment:
should this be --map-col-item-divider instead? and we probably need a fallback as well?

Reply:
Somehow white spaces were tabs instead of white space characters; let's let a linter worry about that stuff in the future. Adding fallback CSS variables and using map variable.